tests/snappi/qos/test_ipip_packet_reorder_with_snappi.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import pytest
+
+from files.packet_reorder_helper import run_ipip_packet_reorder_test
+from tests.common.helpers.assertions import pytest_require
+from tests.common.fixtures.conn_graph_facts import conn_graph_facts,\
+    fanout_graph_facts # noqa F401
+from tests.common.snappi.snappi_fixtures import snappi_api, snappi_testbed_config # noqa F401
+from tests.common.snappi.qos_fixtures import prio_dscp_map # noqa F401
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [pytest.mark.topology('t0', 'tgen')]
+
+
+def test_ip_in_ip_packet_reorder(snappi_api, # noqa F811
+                                 snappi_testbed_config, # noqa F811
+                                 conn_graph_facts, # noqa F811
+                                 fanout_graph_facts, # noqa F811
+                                 duthosts,
+                                 rand_one_dut_hostname,
+                                 rand_one_dut_portname_oper_up,
+                                 enum_dut_lossless_prio,
+                                 prio_dscp_map): # noqa F811
+    """
+    Validate that IPinIP RDMA packets are not being reordered on a single lossless priority
```

Comment:
It seems to me that this test should not be limited to a lossless priority. Instead, if should cover all the priorities.

Reply:
This specific testgap covers a scenario where the packets are lossless

Reply:
> This specific testgap covers a scenario where the packets are lossless

Do we have tests for lossy priorities?

Reply:
We are not concerned with packet re-ordering with lossy priorities

Reply:
> We are not concerned with packet re-ordering with lossy priorities - do not know what the behavior should be in that instance.
 
There should be no packet re-ordering for any priorities, no matter lossy or lossless.